### DISCUSSION TITLE
Turkey Emergency Earthquake Support

### DISCUSSION THREAD

**Discussion by productdevbook** [vercel/community#1461] [Help] [answered]
### Summary

In Turkey, there was an earthquake of more than 7 and it happened 2 times in same day. Tens of thousands of people were injured in this earthquake.

We will collect information from the needy, collect it on a server, and share the data with the relevant and authorized institutions of the Turkish state.

In addition, since each of these software will be opensource and many people will support it, we will also publish it on documentation sites.

Vercel, can you sponsor us for free on the new team building section on their platform?

This team "https://github.com/acildeprem" should integrate all repos and have unlimited plans and unlimited bandwith limits.

My account is "vercel.com/productdevbook". At the moment, I only able to do it for a fee in creating a new team.

After your support and free team building we will enter your requests here in all documentation and websites "https://vercel.com/guides/can-vercel-sponsor-my-open-source-project#sponsorship-criteria".

Thanks.

**Comment by leerob** [comment, marked as answer]
Could you please contact `sponsorships@vercel.com` as the OSS sponsorship guide details? Thank you.

**Reply by productdevbook** [reply]
Thank you i send

**Reply by productdevbook** [reply]
Hi @leerob `sponsorships@vercel.com` no one made a comeback.
